safe_transaction_service/history/serializers.py

```diff
@@ -241,12 +241,16 @@ def save(self, **kwargs):
         safe_address = self.context['safe_address']
         ethereum_client = EthereumClientProvider()
         safe = Safe(safe_address, ethereum_client)
-        try:
-            safe_tx_gas = safe.estimate_tx_gas(self.validated_data['to'], self.validated_data['value'],
-                                               self.validated_data['data'], self.validated_data['operation'])
-        except IOError as exc:
-            raise NodeConnectionException(f'Node connection error when estimating gas for safe {safe_address}') from exc
-        return {'safe_tx_gas': safe_tx_gas}
+        exc = None
+        # Retry twice to get an estimation
+        for _ in range(2):
+            try:
+                safe_tx_gas = safe.estimate_tx_gas(self.validated_data['to'], self.validated_data['value'],
+                                                   self.validated_data['data'], self.validated_data['operation'])
+                return {'safe_tx_gas': safe_tx_gas}
+            except IOError as _exc:
```

Comment:
Should we handle this like you did here: https://github.com/gnosis/safe-transaction-service/pull/383

Reply:
As I'm not using Multicall here I'm not expecting a `ValueError`, but adding it will not harm. Nice catch!

Reply:
Added it